lms/djangoapps/verify_student/services.py

```diff
@@ -101,6 +101,22 @@ def get_verified_users(cls, users):
             ManualVerification.objects.filter(**filter_kwargs).select_related('user')
         )
 
+    @classmethod
+    def get_verified_user_ids(cls, users):
```

Comment:
Can you remove the `get_verified_users` method above if no other code is calling it now?  Thanks.

Reply:
Done, good call.